src/Models/LagrangianParticleTracking/drogued_dynamics.jl

```diff
@@ -0,0 +1,50 @@
+"""
+    DroguedBuoyDynamics(drogue_depths)
+
+`DroguedBuoyDynamics` goes in the `dynamics` slot of `LagrangianParticles` 
+and modifies their behaviour to mimic the behaviour of bouys which are 
+drogued at `drogue_depths`. The particles remain at the their `z` position
+so the "measurment depth can be set", and then are advected in the `x` and `y`
+directions according to the velocity field at `drogue_depths`.
+
+`drogue_depths` should be an array of length `length(particles)`.
+"""
+struct DroguedBuoyDynamics{DD}
+    drogue_depths :: DD
```

Comment:
```suggestion
    depths :: DD
```

Reply:
and if accept this suggestion, modify the docstring as well?